build_tools/bazel_to_cmake/bazel_to_cmake_converter.py

```diff
@@ -121,9 +124,10 @@ def _convert_srcs_block(self, srcs):
         #      ':generated.c' -> 'generated.c'
         # - dropping any leading '//', and internal ':' by '/', as in:
         #      '//path/to/package:generated.c' ->  'path/to/package/generated.c'
+        # TODO: support reference to rules.
```

Comment:
Usual "TODO" style across the project is either

* `TODO(#1234):` when linking to an issue
* `TODO(username):` for the person with the most context on a topic / who would likely implement it